modules/api1/docs.md

```diff
@@ -57,6 +57,32 @@ You may pass additional parameters when querying a list:
   returned.
 - `yiiVersion` - version of the framework project built with. Either `1.0` or `1.1`.
 
+
+### Get primary image <a href="#projects-view-primary-image" id="projects-view-primary-image">#</a>
+
+In order to get primary image of a project, use the following request: 
+
+> GET [/projects/1/primary-image](/en/api/1.0/projects)
+
+In the above `1` is the project ID.
+
+Each image object contains the following fields: 
+- id: image id.
+- thumbnailAbsoluteUrl: absolute url to thumb.
+- updatedAt: UNIX timestamp indicating when image was updated last time.
+
+
+### Update primary image <a href="#projects-update-primary-image" id="projects-update-primary-image">#</a>
```

Comment:
If it `primaryImageID` would be exposed via `/project/1` API then it would make sense to update using `POST /project/1`.